config/default.js

```diff
@@ -350,6 +352,10 @@ module.exports = {
 
   // Feature flags.
   //
+
+  enableFeatureDiscoTaar: false,
+  discoTaarIdCookie: 'taarId',
```

Comment:
Why is this located in the global config if it only applies to the `disco` app?

Reply:
Do you mean both vars or just the cookie? Initially I had this in just disco (default-disco) but I think I got errors when I run amo:dev. I'll recheck this tho...